hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallLocalSuite.java

```diff
@@ -285,7 +285,6 @@ private HapiSpec erc20Query() {
     }
 
     // https://github.com/hashgraph/hedera-services/pull/5485
-    @HapiTest
     private HapiSpec callLocalDoesNotCheckSignaturesNorPayer() {
```

Comment:
Disabling this test because it's failing because it's failing in mono as well. My theory is that it was succeeding until now because the throttles were not implemented.

@MiroslavGatsanoga noticed that this test is part of `ConcurrentSuites` and it uses `throttles-dev.json` and it's running on a single node - I tried that but it's still failing.